region/brinstar/kraid/Warehouse Zeela Room.json

```diff
@@ -175,6 +175,85 @@
       "name": "Base",
       "requires": []
     },
+    {
+      "link": [1, 2],
+      "name": "Come in Shinecharging, Leave With Spark",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 5,
+          "openEnd": 0
+        }
+      },
+      "requires": [
+        {"or": [
+          {"enemyKill": {
+            "enemies": [["Zeela"]],
+            "explicitWeapons": ["Wave", "Spazer", "Plasma", "Missile", "Super"]
+          }},
+          {"and": [
+            "canKago",
+            {"enemyDamage": {
+              "enemy": "Zeela",
+              "type": "contact",
+              "hits": 1
+            }}
+          ]}
+        ]},
+        "canShinechargeMovementTricky",
+        {"shinespark": {"frames": 5}}
+      ],
+      "exitCondition": {
+        "leaveWithSpark": {}
+      },
+      "unlocksDoors": [
+        {"types": ["super"], "requires": []},
+        {"types": ["missiles", "powerbomb"], "requires": ["never"]}
+      ],
+      "note": [
+        "Gain a shinecharging while entering the room, and angle diagonally down to shoot the block.",
+        "While falling, kill the Zeela or perform a Kago through it.",
+        "Quickly reach the door below and spark out."
+      ]
+    },
+    {
+      "link": [1, 2],
+      "name": "Come in Shinecharged, Leave With Spark",
+      "entranceCondition": {
+        "comeInShinecharged": {
+          "framesRequired": 165
+        }
+      },
+      "requires": [
+        {"or": [
+          {"enemyKill": {
+            "enemies": [["Zeela"]],
+            "explicitWeapons": ["Wave", "Spazer", "Plasma", "Missile", "Super"]
+          }},
+          {"and": [
+            "canKago",
+            {"enemyDamage": {
+              "enemy": "Zeela",
+              "type": "contact",
+              "hits": 1
+            }}
+          ]}
+        ]},
+        "canShinechargeMovementTricky",
+        {"shinespark": {"frames": 4}}
+      ],
+      "exitCondition": {
+        "leaveWithSpark": {}
+      },
+      "unlocksDoors": [
+        {"types": ["super"], "requires": []},
+        {"types": ["missiles", "powerbomb"], "requires": ["never"]}
+      ],
+      "note": [
+        "Enter with a shinecharge, jumping and aiming down and shoot the block.",
```

Comment:
The fastest way I found was to
Jump & Shoot shot block->Morph -> Neutral DBoost on bottom Zeela-> bump ceiling/floor ->unmorph+downback
Still, 150 frames.

Reply:
Interesting, sounds like this could be a separate strat with lower frame count but an additional item requirement (Morph). I wasn't able to get it yet so for now I just put in a FIXME; you could add it if you like.